data/changelog.txt

```diff
@@ -511,6 +521,11 @@ Crafting GUI Search: cancel the search with ESC
 use the monospaced font for the ascii art on the end screen
 Increased available space for skills chapter in description tab of character creation menu
 ImGui inherits theme colors
+Sidebar widget to show faction territory
+Revert horizontal separator color back to dark gray
```

Comment:
@Night-Pryanik hey Boss, these really need to be copy-edited by someone before merge. This is the abbreviated changelog that should only contain notable changes. This revert for instance shouldn’t have been included.